web/src/lib/domain-logic/getVisibleSideBarCards.ts

```diff
@@ -0,0 +1,13 @@
+import { SidebarCardContent } from "@/lib/types/types";
```

Comment:
I don't know if I agree with some of these functions being added to `/domain-logic` this still feels pretty component specific to me.

Reply:
moved related helpers to a single function, and created this ticket - https://www.pivotaltracker.com/story/show/187105804